src/modules.js

```diff
@@ -458,7 +458,8 @@ var LibraryManager = {
       'library_glew.js',
       'library_html5.js',
       'library_signals.js',
-      'library_async.js'
+      'library_async.js',
+      'library_vr.js'
```

Comment:
Oh, I wonder if we should start breaking this pattern now, and not add new libraries to the link list by default. We have been talking that these built-in libraries would get included only if user would pass a linker flag `-lvr.js`, as that would make it cleaner not to attempt to link in all the code all the time. What do you think?